packages/now-cli/src/commands/alias/set.ts

```diff
@@ -225,36 +205,31 @@ export default async function set(
     return 1;
   }
 
-  // Assign the alias for each of the targets in the array
-  for (const target of targets) {
-    output.log(`Assigning alias ${target} to deployment ${deployment.url}`);
+  output.log(`Assigning alias ${aliasTarget} to deployment ${deployment.url}`);
```

Comment:
I see this message `Assigning alias` twice, once here and once on line 139

Is that correct?

Reply:
The first message is for a single argument in `now alias set <argument>`